etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messages/SendMessageHelper.java

```diff
@@ -21,67 +21,66 @@ public static SendMessageHelper getInstance() {
 
     private SendMessageHelper() {}
 
-    public void savePartnerMetadataForReceivedMessage(PartnerMetadata partnerMetadata) {
-        if (partnerMetadata.receivedSubmissionId() == null) {
+    public void savePartnerMetadataForInboundMessage(PartnerMetadata partnerMetadata) {
+        if (partnerMetadata.inboundReportId() == null) {
             logger.logWarning(
-                    "Received submissionId is null so not saving metadata for received message");
+                    "Inbound reportId is null so not saving metadata for received message");
```

Comment:
There are few places with `Inbound reportId` or `Outbound reportId`. I think it's better to just rename them as `InboundReportId`/`OutboundReportId`, without the whitespace. That way it'd be easier search

Reply:
Good point. I swapped them out